perf/security/sds-tests/workload-sds/setup_test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+NUM=${NUM:?"specify the number of httpbin and sleep workloads"}
+CLUSTER=${CLUSTER:?"specify the cluster for running the test"}
+
+if ! istioctl version; then 
```

Comment:
Maybe adding an output "istioctl is not installed or invalid istioctl version" here so the user knows why the test exits?

Reply:
It is not necessary to add this, ``set -ex`` dumps all the commands being executed. If istioctl does not work, the output looks like this:

```
$ NUM=2 CLUSTER=kind-istio-testing ./setup_test.sh 
+ NUM=2
+ CLUSTER=kind-istio-testing
+ istioctl version
./setup_test.sh: line 21: istioctl: command not found
+ exit
```